package.json

```diff
@@ -2,14 +2,15 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "build": "lerna run build",
+    "build-snowpack": "cd packages/snowpack && yarn build",
     "publish": "lerna publish",
     "format": "prettier --write \"packages/*/src/**/*.ts\"",
-    "setupTest": "yarn build && yarn --force --ignore-engines && lerna run testbuild",
-    "test": "npm run setupTest && jest --test-timeout=15000"
+    "setupTest": "yarn build-snowpack && yarn --force --ignore-engines && lerna run testbuild",
```

Comment:
`lerna run *` will run a command in every repo. We’re using `lerna run testbuild` to do all of our build comparisons efficiently.

However, in all our templates, we have a `"build"` script we really don’t need to run for our test suite. In fact, it slows everything down. Currently, we really only need `snowpack` to build for tests and nothing else. So this change reflects that.